tests/ci/version_helper.py

```diff
@@ -66,13 +66,27 @@ def _get_version_from_line(line):
     _, ver_with_bracket = line.strip().split(' ')
     return ver_with_bracket[:-1]
 
+def get_tweak_from_git_describe(repo_path):
+    # something like v21.12.1.8816-testing-358-g81942b8128
+    # or v21.11.4.14-stable-31-gd6aab025e0
+    output = subprocess.check_output(f"cd {repo_path} && git describe --long", shell=True).decode('utf-8')
+    commits_number = int(output.split('-')[2])
```

Comment:
I'd take the second from the tail. The tag _may_ contain hyphens
```suggestion
    commits_number = int(output.split('-')[-2])
```